metadata.yaml

```diff
@@ -21,3 +21,7 @@ resources:
     type: oci-image
     description: OCI image for prometheus-edge-hub
     upstream-source: facebookincubator/prometheus-edge-hub:1.1.0
+
+provides:
+  monitoring:
```

Comment:
Oh gawd not “monitoring”! Monitoring is not equivalent to collecting metrics. It is the act of collecting telemetry, of which metrics is just one type :-)

please rename to “metrics-endpoint”, it works very well with the “provide” :-)

Reply:
For the record, a very reputable source agrees with me about the meaning of “monitoring”: https://ubuntu.com/observability/what-is-observability :P

Reply:
cool read, I didn't know the difference tbh :)

Reply:
Agreed, fixed